annotation-utils/src/main/java/edu/isi/vista/annotationutils/Rehydrate.kt

```diff
@@ -0,0 +1,75 @@
+package edu.isi.vista.annotationutils
+
+import edu.isi.nlp.io.OffsetIndexedCorpus
+import edu.isi.nlp.symbols.Symbol
+import edu.isi.nlp.io.DocIDToFileMappings
+import com.google.common.base.Optional
+import com.fasterxml.jackson.databind.ObjectMapper
+import edu.isi.nlp.parameters.Parameters
+import java.io.File
+
+/**
+ * Puts the document text back into the 'sofaString' field inside json files created by `ExportAnnotations.kt`
```

Comment:
Explain where and why they were removed

Reply:
Also mention this program in the repo's `README.md`

Reply:
Also, while we use the "dehydrate/rehydrate" language informally, it would be clearer to future readers if we called this something like `RestoreDocumentText`

Reply:
Still need to rename away from re-hydrate